java/integrated-windows-authentication/README.md

```diff
@@ -1,32 +1,47 @@
-# Integrated Windows Authentication
+# Integrated windows authentication
```

Comment:
I can't really test this one unfortunately, and some of the stuff here is running behind the scenes, but I trust the API is all good!